exampleApp/.env.mysql

```diff
@@ -0,0 +1,9 @@
+DB_USERNAME=seed
+DB_PASSWORD=Ge0rgesMoustaki
+DB_DATABASE=seed
+DB_PORT=5434
+# don't use DB for sessions during tests
+SESSION_DB_URL=false
+TZ=UTC
+DB_TYPE=postgres
+DB_ HOST=localhost
```

Comment:
```suggestion
DB_HOST=localhost
```

Reply:
These are not the credentials you specify in the .circleci/config, why does it work?